Tests/MoyaTests/TestPlugin.swift

```diff
@@ -12,6 +12,12 @@ final class TestingPlugin: PluginType {
         request.addValue("yes", forHTTPHeaderField: "prepared")
         return request
     }
+    
+    func prepare(_ request: URLRequest, multipartBody: [MultipartFormData], target: TargetType) -> (URLRequest, [MultipartFormData]) {
+        var request = request
+        request.addValue("yes", forHTTPHeaderField: "prepared-multipart-body")
+        return request
```

Comment:
This is failing to compile because the return type is a tuple and you are returning a request :)
```suggestion
        return (request, multipartBody)
```

Reply:
Thank you.